salt/vault/map.jinja

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+{% import_yaml "vault/maps/defaults.yaml" or {} as defaults %}
+{% import_yaml "vault/maps/osfamilymap.yaml" or {} as osfamilymap %}
+{% import_yaml "vault/maps/initfamilymap.yaml" or {} as initfamilymap %}
+
+{%- set merged_defaults = salt.grains.filter_by(defaults,
+    default='vault',
+    merge=salt.grains.filter_by(osfamilymap, grain='os_family',
+      merge=salt.grains.filter_by(initfamilymap, grain='init',
+        merge=salt.grains.get('vault', default={})
+      )
+    )
+) %}
+
+{#- Merge the vault pillar #}
+{%- set vault = salt.pillar.get('vault', default=merged_defaults, merge=True) %}
```

Comment:
salt convention is to namespace formula lookups in pillar using `<name>:lookup`. this helps avoid collisions in the pillar dictionary, in case the user is already using the formula <name> key for something...

```suggestion
{%- set vault = salt.pillar.get('vault:lookup', default=merged_defaults, merge=True) %}
```

Reply:
Don't need to use pillar any more. I was just testing it out. Removing it from the solution.